ws_web/staff_views.py

```diff
@@ -160,6 +162,7 @@ def list(self, request, *args, **kwargs):
 
     def create(self, request, *args, **kwargs):
         data = request.data
+        print(data)
```

Comment:
deleting this?

Reply:
No I want that! This gives us a complete history for a user, which I am hoping we can use to figure out what edge case is leading to the problem

Reply:
this is outputting in Heroku which will flush the log I assume?